docs/technical_reference/unit_models/electrolyzer.rst

```diff
@@ -0,0 +1,245 @@
+
+Degrees of Freedom
+------------------
+For a given electrolyzer design, there are 8 degrees of freedom in addition to the inlet state variables (i.e., temperature, pressure, component flowrates) that should be fixed for the model to be fully specified. In typical design cases the following 8 variables are fixed:
+
+   * membrane current density, :math:`i_{mem}`
+   * anode current density, :math:`i_{ano}`
+   * anode overpotential, :math:`\eta_{ano}`
+   * cathode current density, :math:`i_{cat}`
+   * cathode overpotential, :math:`\eta_{cat}`
+   * current, :math:`I`
+   * current efficiency, :math:`\theta_{current}`
+   * voltage efficiency, :math:`\theta_{voltage}` *or* resistance, :math:`R`
+
+Additionally, the electrolysis reactions at the anode and cathode must be specified with stoichiometry and electrochemical potential (Phillips, 2017). By default, the following stoichiometric variables will be fixed to 0 when the electrolyzer model block is constructed. Therefore, only the nonzero stoichiometry must be additionally specified. As an example for the chlor-alkali process, the following reactions occur with sodium ions permeating from the anolyte to catholyte. The stoichiometry is normalized to 1 mole of electrons as intended in the model framework.
+
+Anode:
+
+    .. math::
+
+        & Cl^-\to{\small\frac{1}{2}}Cl_2+e^-\hspace{4em}E=1.21V \\\\
+
+Cathode:
+
+    .. math::
+
+        & H_2O+e^-\to{\small\frac{1}{2}}H_2+OH^-\hspace{4em}E=-0.99V \\\\
+
+The following variables should then be fixed:
+
+.. code-block::
+
+   # membrane properties
+   m.fs.electrolyzer.membrane_ion_transport_number["Liq", "NA+"].fix(1)
+
+   # anode properties
+   # Cl- --> 0.5 Cl2 + e-
+   m.fs.electrolyzer.anode_electrochem_potential.fix(1.21)
+   m.fs.electrolyzer.anode_stoich["Liq", "CL-"].fix(-1)
+   m.fs.electrolyzer.anode_stoich["Liq", "CL2-v"].fix(0.5)
+
+   # cathode properties
+   # H20 + e- --> 0.5 H2 + OH-
+   m.fs.electrolyzer.cathode_electrochem_potential.fix(-0.99)
+   m.fs.electrolyzer.cathode_stoich["Liq", "H2O"].fix(-1)
+   m.fs.electrolyzer.cathode_stoich["Liq", "H2-v"].fix(0.5)
+   m.fs.electrolyzer.cathode_stoich["Liq", "OH-"].fix(1)
+
+Model Structure
+------------------
+The electrolyzer model consists of 2 ``ControlVolume0DBlocks``, one for the anolyte and another for the catholyte. Currently, the generation of species via electrolysis reactions is handled by the ``custom_molar_term`` within the ``ControlVolume0DBlock``. Coupling this custom molar term with the Faradaic conversion, the material balance may be written by the following and calculated within the ``ControlVolume0DBlock``. Here, membrane flux is handled by relating the stoichiometric coefficient required to satisfy the charge balance. The direction of membrane flow is calculated from anode to cathode in the unit model, lending to the signage of the membrane stoichiometric coefficient shown.
+
```

Comment:
These language isn't quite convention electrochemical statement and a little hard to get by me. For example, membrane flux of what component? what is "stoichiometric coefficient required to satisfy the charge balance."? what is "membrane flow"?       Some overall suggestions are 1) in a single model, you should have one unified flux direction and assign it to be positive, 2) be very careful with the current direction, which may be the same or opposite to the flux direction depending on the charge carrier's +/-; 3) herein you may simply state the stoich coef. is for the mass change on the electrode surface and the t_mem is for the mass transfer across the membrane.